discovery-infra/test_infra/utils.py

```diff
@@ -829,3 +830,32 @@ def get_assisted_controller_status(kubeconfig):
 
     log.info(f'{response.stdout}')
     return response.stdout
+
+
+def _render_socket_endpoint(ip, port):
```

Comment:
move load balancer code to load_balancer_controller maybe?

Reply:
Possible